README.md

```diff
@@ -79,15 +79,15 @@ Note that all instance methods throw an error when something goes wrong or when
 
 **SMTPClient.prototype.connect({timeout})**:Promise;
 
-> Connects to the SMTP server and starts socket I/O activity.
+> Connects to the SMTP server and starts to socket I/O activity.
```

Comment:
A socket is one endpoint of a two-way communication link between two programs running on the network. 
The "socket I/0 activity" is correct.